openedx_learning/apps/authoring/publishing/api.py

```diff
@@ -771,6 +774,64 @@ def create_container_version(
     return container_version
 
 
+class ChildrenEntitiesAction(Enum):
```

Comment:
Do we persist this data structure anywhere, or is it just used during API calls?

Reply:
No it is just used in API calls.